pkg/acsengine/const.go

```diff
@@ -105,7 +105,13 @@ const (
 	// DefaultEtcdVersion specifies the default etcd version to install
 	DefaultEtcdVersion = "3.2.16"
 	// DefaultEtcdDiskSize specifies the default size for Kubernetes master etcd disk volumes in GB
-	DefaultEtcdDiskSize = "128"
+	DefaultEtcdDiskSize = "256"
+	// EtcdDiskSizeGT3Nodes = size for Kubernetes master etcd disk volumes in GB if > 3 nodes
+	EtcdDiskSizeGT3Nodes = "512"
+	// EtcdDiskSizeGT10Nodes = size for Kubernetes master etcd disk volumes in GB if > 10 nodes
+	EtcdDiskSizeGT10Nodes = "1024"
+	// EtcdDiskSizeGT20Nodes = size for Kubernetes master etcd disk volumes in GB if > 20 nodes
+	EtcdDiskSizeGT20Nodes = "2048"
```

Comment:
Here I think it would make sense to change naming to be `DefaultEtcdDiskSize<Insert_Number_of_Nodes>` since all of them are defaults that can be overridden by the user if I understand correctly. The name EtcdDiskSizeGT20Nodes + the comment makes it seem like it's static